### PR TITLE
[RDY] nvim.1: Misc. improvements

### PR DESCRIPTION
**Pull request by ghost**
From the commit message:

```
Prefer typographic single quotes instead of straight double quotes.
This avoids confusion when double quotes are used as part of arguments.

Annotate EOF as a "defined variable".
Assuming mandoc is used, `apropos Dv=EOF` should now work.

Clarify in parentheses what a hyphen is, as users might only know it
as a "dash". The same was already done for colons.

Group environment variables more logically.
```

This was tested with groff 1.22.3 in addition to mandoc from OpenBSD current, although I'm not using any new macros from those systems so I doubt it matters.

### CODE REVIEW

**Comment by ianks** [comment]
LGTM

**Comment by Pyrohh** [comment]
Thanks, I'll merge this after one more review. @fmoralesc?

File: man/nvim.1 [hunk 1]
```diff
@@ -345,6 +351,11 @@ Print version information and exit.
 .El
 .Sh ENVIRONMENT
 .Bl -tag -width Fl
+.It Ev VIM
+Used to locate various user files, such as the user-local nvimrc.
```
Comment (fmoralesc) [inline man/nvim.1]: I think just "user vimrc" is a bit better (didn't find the expression "user-local vimrc" anywhere else in the repo, and it feels weird to me).

Comment (Pyrohh) [inline man/nvim.1]: Sure, I'll do that. I took the existing phrasing out of the tarsnap manuals, so maybe I was trying to hard to sound professional ;)

**Comment by Pyrohh** [comment]
Repushed.

**Comment by fmoralesc** [comment]
LGTM. Very professional, yet not intimidating ;)

**Comment by Pyrohh** [comment]
Thanks both.

> Very professional, yet not intimidating ;)

```
$ man fmorales-editor
FMORALES-EDITOR(1)           General Commands Manual                  FMORALES-EDITOR(1)
less: Trailing whitespace detected, fatal error
$
```

**Pull request closed without merging**